docs/src/tutorials/aws-example-policy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "iot:Connect",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:client/${iot:ClientId}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/messages/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Subscribe",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topicfilter/thinedge/devices/${iot:ClientId}/messages/devicebound/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Receive",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/messages/devicebound/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:RetainPublish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/tedge/health/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/tedge/health/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/test-connection"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Receive",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/test-connection"
```

Comment:
I would expect `connection-success` here. Isn't it?

Reply:
`connection-success` is not required in the policy as this topic is only used locally. The idea behind the connection check is to publish a message on the `connection-test` topic which is forwarded on to the cloud. We then subscribe from the client to the `connection-test` topic on AWS and map the messages to the `connection-success` topic.